gig/models.py

```diff
@@ -147,7 +135,7 @@ def member_plans(self):
         for a in absent:
             Plan.objects.create(gig=self, assoc=a, section=s)
         # now that we have one for every member, return the list
-        return self.plans
+        return getattr(self, 'plans') # technically the gig object has no 'plans' attribute; it's defined by the ORM system
```

Comment:
I think the pylint error here was not due to the fact that it couldn't track the foreign key, but because it's Gig, and not AbstractGig, that Plan maps to.  At the moment, this will never be a problem.  But if we get another descendant of AbstractGig, this could be a bug.